public/js/routes.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+var app = app || {};
+
+page('/', () => { /index });
```

Comment:
I think this might be what is breaking your code.

Reply:
added app.Project.initIndexPage instead of whats in the { }.